bin/dotd

```diff
@@ -415,6 +417,15 @@ def print_database_changes(pull_request_number)
   EOS
 end
 
+# if the commit is more than 24 hours old, warn
+def print_old_commit(commit_sha)
```

Comment:
nit: My preference would be to name the method the self-descriptive `warn_if_old_commit`.